hs_access_control/management/commands/set_shareable.py

```diff
@@ -0,0 +1,63 @@
+"""Set the shareable bit for a resource to True or False
+   This is a workaround for the fact that published, unshareable i
+   resources can't be added to collections.
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+
+
+def check_shareable(rid, options):
+    try:
+        resource = get_resource_by_shortkey(rid, or_404=False)
+    except BaseResource.DoesNotExist:
+        print("{}: does not exist".format(rid))
+        return
+
+    print("{}: shareable bit is now {}".format(rid, resource.raccess.shareable))
+    if options['on'] and not options['off']:
+        print("{}: changing sharable bit to True".format(rid))
+        resource.raccess.shareable = True
+        resource.raccess.save()
+    elif options['off'] and not options['on']:
+        print("{}: changing sharable bit to False".format(rid))
+        resource.raccess.shareable = False
+        resource.raccess.save()
+    elif options['off'] and options['on']:
```

Comment:
if you put this first, you'd only have to check on or off instead of both in each if conditional above.

Reply:
Done.